digiroad2-api-common/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Context.scala

```diff
@@ -91,6 +91,13 @@ class DynamicAssetUpdater(dynamicAssetService: DynamicLinearAssetService) extend
   }
 }
 
+class RoadWorksAssetUpdater(roadWorkAssetService: RoadWorkService) extends Actor {
```

Comment:
Is a separate actor needed for this asset? The actor for dynamic linear assets also calls updateChangeSet

Reply:
Yes is needed, because on RoadWorkService we override the enrichPersistedLinearAssetProperties method and if we want to use him instead of enrichPersistedLinearAssetProperties of DynamicLinearAssetService we need to keep this.